### DISCUSSION TITLE
Questions about downloading

### DISCUSSION THREAD

**Discussion by Vzz1c** [ScoopInstaller/Scoop#5194] [Help and Questions] [answered]
extras/gpu-z download path is
http://us2-dl.techpowerup.com/files/GPU-Z.2.50.0.exe
In fact this link is not normally downloadable, the real download link usually needs to be clicked to get an address, if you use this address you need to prefix it:
User-Agent: Scoop/1.0 (+http://scoop.sh/) xxxx/xxxx
Is the target site letting this go? Or did the scoop command do something? How can I be sure that this download is possible when I encounter a similar situation?

**Comment by rashil2000** [comment, marked as answer]
That website has whitelisted Scoop - it will not allow anyone else to use the direct link
